docs/content/en/docs/getting-started/production-environment/gs-mgmt-cluster.md

```diff
@@ -0,0 +1,279 @@
+     flux:
+       github:
+         owner: exampleuser
+         repository: "flux-example"
+         personal: true
+   ---
+         # The rest of this is for the management cluster only
+   apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+   kind: VSphereMachineConfig
+   metadata:
+     name: sigb-etcd
+   spec:
+     datastore: "/SDDC-Datacenter/datastore/WorkloadDatastore"
+     diskGiB: 25
+     folder: "/SDDC-Datacenter/vm/capv/sigb"
+     memoryMiB: 8192
+     numCPUs: 2
+     osFamily: ubuntu
+     template: "/SDDC-Datacenter/vm/Templates/ubuntu-2004-kube-v1.21.2"
+     resourcePool: "*/Resources/Compute-ResourcePool"
+     users:
+       - name: capv
+         sshAuthorizedKeys:
+           - ssh-rsa AAAAB3... sigb@xyz.ant.example.com 
+   ```
+
+    Further information about the values in the `eksa-mgmt-cluster.yaml` can be found in the [cluster specification
+    reference]({{< relref "../../reference/clusterspec/vsphere.md" >}})
+
+1. Set Credential Environment Variables
+
+   Before you create a workload cluster, you will need to set and export these environment variables for your vSphere user name and password.
+Make sure you use single quotes around the values so that your shell does not interpret the values:
+   
+   ```bash
+   export EKSA_VSPHERE_USERNAME='billy'
+   export EKSA_VSPHERE_PASSWORD='t0p$ecret'
+   ```
+
+1. Create a management cluster
+
+   After you have created your `eksa-mgmt-cluster.yaml` and set your credential environment variables, you will be ready to create a management cluster:
+   ```bash
+   eksctl anywhere create cluster -f eksa-mgmt-cluster.yaml
+   ```
+   Example command output
+   ```
+    Performing setup and validations
+    ✅ Connected to server
+    ✅ Authenticated to vSphere
+    ...
+    🎉 Cluster created!
+   ```
+
+1. Once the management cluster is created you can use it with the generated `KUBECONFIG` file in your local directory:
+
+   ```bash
+   export KUBECONFIG=${PWD}/${CLUSTER_NAME}/${CLUSTER_NAME}-eks-a-cluster.kubeconfig
+
+   ```
+1. Check the management cluster:
+
+   To check that the cluster completed, list the machines to see the control plane, etcd, and worker nodes:
+
+   ```bash
+   kubectl get machines -A
+   ```
+
+   Example command output
+   ```
+   NAMESPACE   NAME                PROVIDERID        PHASE    VERSION
+   eksa-system sigb-b2xyz          vsphere:/xxxxx    Running  v1.21.2-eks-1-21-5
+   eksa-system sigb-etcd-r9b42     vsphere:/xxxxx    Running  
+   eksa-system sigb-md-8-6xr-rnr   vsphere:/xxxxx    Running  v1.21.2-eks-1-21-5
+   ...
+   ```
+
+   To ensure you are looking at the management cluster, list the CRD to see that the name of its management cluster is itself:
+
+   ```bash
+   kubectl get clusters sigb -o yaml
+   ```
+
+   Example command output
+   ```
+   ...
+   kubernetesVersion: "1.21"
+   managementCluster:
+     name: sigb
+   workerNodeGroupConfigurations:
+   ...
+
+1. Modify the workload cluster config (`eksa-w01-cluster.yaml`):
```

Comment:
Do we not have a CLI command to generate a workload cluster config? That would be really helpful.

Reply:
My understanding is that it is not much different than the management cluster yaml file. But I'll ask @mdsgabriel about it.

Reply:
> Do we not have a CLI command to generate a workload cluster config? That would be really helpful.
no. the same spec can be used to crate a management cluster or a workload cluster.

if the optional flag --kubeconfig is passed to the create command then the cli will create a workload cluster